helpers/all.js

```diff
@@ -3,25 +3,19 @@
 const _ = require('lodash');
 
 /**
- * Yield block only if all arguments are valid
+ * Yield block only if all args are valid
  *
  * @example
  * {{#all items theme_settings.optionA theme_settings.optionB}} ... {{/all}}
  */
 const factory = () => {
-    return function() {
-        var args = [], opts, result;
+    return function(...args) {
 
```

Comment:
Still learning handlebars so this may be an invalid question.  Why do we need to return a function, could we do something like this: `const factory = (...args) => { <logic>  } `

Reply:
That's something, I didn't look into but don't arrow functions lose their this context?

Reply:
I tested the code using that and it can't find all anymore so no we can not do something like that.

Reply:
Sorry I didn't follow, could you clarify what happened when you tested it?

Reply:
Actually, I realized that I had to fix some things due to update of lodash version.  Tests fail with opts.fn is not a function if I change things to be like what was stated previously.